### PR TITLE
FIX : Make the switcher optional

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
Change the default switcher. Make the navigation end bar switcher only if it is present.

### CODE REVIEW

File: doc/source/conf.py [hunk 1]
```diff
@@ -60,10 +59,10 @@
...
         },
     ],
-    "switcher": {
-        "json_url": f"https://{cname}/release/versions.json",
-        "version_match": get_version_match(__version__),
-    },
+    # "switcher": {
+    #    "json_url": f"https://{cname}/release/versions.json",
+    #    "version_match": get_version_match(__version__),
+    # },
```
Comment (RobPasMue) [inline doc/source/conf.py:65]: Do not leave commented out code. If it is no longer needed, delete it.
```suggestion
```

File: doc/source/conf.py [hunk 2]
```diff
@@ -11,14 +11,13 @@
 
 LaTeXBuilder.supported_image_types = ["image/png", "image/pdf", "image/svg+xml"]
 
-from ansys_sphinx_theme import (
+from ansys_sphinx_theme import (  # get_version_match,
```
Comment (RobPasMue) [inline doc/source/conf.py:14]:
```suggestion
from ansys_sphinx_theme import (
```
Same here

**Review by RobPasMue — CHANGES_REQUESTED** [review]

File: doc/source/conf.py (hunk 1 above)
Comment (Revathyvenugopal162) [reply doc/source/conf.py:65]: Yes, i was testing it, to check it is failing or not.

**Review by RobPasMue — APPROVED** [review]
Sounds reasonable. Thanks @Revathyvenugopal162

**Pull request merged**